File: C#_Aplication/Agilize/Projects.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Agilize
{
    public class Projects
    {
        public String projectOwner {get; set;}
        public String projectName {get; set;}
        public BindingList<Users> arrayProjectUsers { get; set; }

        public BindingList<Tasks> arrayTasks { get; set; }
        public Projects() { }

        public Projects(string projectOwner, String projectName, BindingList<Users> arrayProjectUsers, BindingList<Tasks> arrayTasks)
        {
            arrayTasks = new BindingList<Tasks>();
            arrayProjectUsers = new BindingList<Users>();
            this.projectOwner = projectOwner;
            this.projectName = projectName;
            this.arrayProjectUsers = arrayProjectUsers;
            this.arrayTasks = arrayTasks;
        }

        override public String ToString()
        {
            return projectName.ToString();
        }
    }
}

File: Mobile_Aplication/app/src/main/java/com/example/agilize/classes/Projects.kt
package com.example.agilize.classes

import android.os.Parcel
import android.os.Parcelable


class Projects(var projectOwner:String, var projectName: String, var arrayProjectUsers: MutableList<Users>,
               var  arrayTasks: MutableList<Tasks>): Parcelable {

    constructor(parcel: Parcel) : this(
        parcel.readString().toString(),
        parcel.readString().toString(),
        mutableListOf<Users>().apply {
            parcel.readTypedList(this, Users.CREATOR)
        },
        mutableListOf<Tasks>().apply {
        parcel.readTypedList(this, Tasks.CREATOR)
        }
    )

    override fun writeToParcel(parcel: Parcel, flags: Int) {
        parcel.writeString(projectOwner)
        parcel.writeString(projectName)
        parcel.writeTypedList(arrayProjectUsers)
        parcel.writeTypedList(arrayTasks)
    }

    override fun describeContents(): Int {
        return 0
    }

    companion object CREATOR : Parcelable.Creator<Projects> {
        override fun createFromParcel(parcel: Parcel): Projects {
            return Projects(parcel)
        }

        override fun newArray(size: Int): Array<Projects?> {
            return arrayOfNulls(size)
        }
    }
}